config/initializers/rack_attack.rb

```diff
@@ -141,7 +141,8 @@ class Rack::Attack::Request
     period: 1.second,
   ) do |request|
     if request.tracking_enabled? && request.anonymous?
-      request.request_ip
```

Comment:
looks like if we don't exclude history reqs here, the history req is not throttled as it superseded by the more general brute force throttle